galleries/users_explain/axes/axes_units.py

```diff
@@ -0,0 +1,278 @@
+"""
+.. _user_axes_units:
```

Comment:
The first box is the quick start guide.  This support could/should be mentioned there (if its not already), but no where near in this detail.

Reply:
> I think the general units stuff should be a sentence or so here and fleshed out in the units tutorial that I think

There is a sentence at the top that says: 

```
The method to add converters to Matplotlib is described in `matplotlib.units`.
Here we briefly overview the built-in date and string converters.
```

The final section is because people are quite likely to see more general units support in other libraries or even in our examples.  I would like to keep it.

Reply:
>  I don';t think folks would think to look for it in Axes cause this is a data thing and everything else is layout or labeling

I disagree that this is not about labelling - that is the major advantage of unit support, providing useful tick locators and formatters.

Reply:
> Yes, but using unitid data hands over control of locators and formatters to the unit implementation. The users interaction with the unit framework is in the data they plot, not in the ticks/labels (annotations) they set.

After passing unitful data to the plotting routines, which the user doesn't control, the only way to interact with the units machinery is via tick locators, formatters, and axis limits.  Indeed these are the things that cause confusion, and the main motivation for this section.  eg. user uses mplfinances converter which ignores weekends, and then tries to use our Locators and Formatters and get nonsense results.  I'd say a _lot_ of people who use datetimes end up setting the Locator and customizing the Formatters.